common4j/src/main/com/microsoft/identity/common/java/logging/Logger.java

```diff
@@ -284,6 +333,27 @@ public static void info(final String tag,
         log(tag, Logger.LogLevel.INFO, null, message, null, false);
     }
 
+    /**
+     * Send a {@link Logger.LogLevel#INFO} log message. The object to printed may contain PII,
+     * depending on its containsPii() value. If isAllowPii() is set to false, a PII-safe string
+     * representation of the object will be used. If allowPII() is set to true, an un-sanitised
+     * string representation of the object will be used.
+     *
+     * @param tag     Used to identify the source of a log message. It usually identifies the class
+     *                or activity where the log call occurs.
+     * @param message The message to log.
+     * @param object The object to be printed.
+     */
+    public static void infoWithObject(final String tag,
+                            final String message,
+                            final ILoggable object) {
```

Comment:
Do we need public static void infoWithObject(final String tag, **final String correlationID**, final String message, final ILoggable object) like warnWithObject

Reply:
There are no usages of this method yet, so I didn't add it. You may have to add it with your "clean up logging" PR - is that okay?